backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/cfg/ElasticsearchBackendSettings.java

```diff
@@ -267,7 +267,6 @@ private Defaults() {
 
 		public static final List<String> HOSTS = Collections.singletonList( "localhost:9200" );
 		public static final String PROTOCOL = "http";
-		public static final int REQUEST_TIMEOUT = 60000;
```

Comment:
You should also update the javadoc on `ElasticsearchBackendSettings#REQUEST_TIMEOUT`:

```
	 * Defaults to {@link Defaults#REQUEST_TIMEOUT}.
```

=>

```
	 * Defaults to no request timeout.
```